src/components/JobWizard/JobWizard.js

```diff
@@ -328,7 +328,7 @@ const JobWizard = ({
         })
         .then(() => {
           return navigate(
-            `/projects/${params.projectName}/jobs/${isSchedule ? SCHEDULE_TAB : MONITOR_JOBS_TAB}`
+            `/projects/${params.projectName}/jobs/${isSchedule ? SCHEDULE_TAB : MONITOR_JOBS_TAB}${window.location.search}`
```

Comment:
`search` parameters must be added only if we navigate to the same page that we were before. Otherwise wrong filters will be applied